hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/initialization/BackedSystemAccountsCreator.java

```diff
@@ -82,11 +87,14 @@ public void ensureSystemAccounts(
             if (accounts.contains(id)) {
                 continue;
             }
+            final HederaAccount account;
             if (num == accountNums.treasury()) {
-                accounts.put(id, accountWith(tinyBarFloat, expiry));
+                account = accountWith(tinyBarFloat, expiry);
             } else {
-                accounts.put(id, accountWith(0, expiry));
+                account = accountWith(ZERO_BALANCE, expiry);
             }
+            accounts.put(id, account);
+            systemAccountsCreated.add(account);
```

Comment:
Does this line `systemAccountsCreated.add(account);` also need to be added at line 111, for the node rewards and staking rewards accounts?

Reply:
@kimbor  Thanks !  You are correct , but we already export records for creation of 0.0.800 and 0.0.801 [here on network reset](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/migration/MigrationRecordsManager.java#L125)

Reply:
Ok